catkin_ws/src/sensors/src/depth_republish.cpp

```diff
@@ -0,0 +1,48 @@
+#include <ros/ros.h>
```

Comment:
this guy did not read my c++ formatting wiki page lol

Reply:
im not clear on what the issue is with that

Reply:
I'm just saying as a joke, i don't really mind it. But we do have a c++ formatting guide on the wiki based on Google's programming standards